src/main/java/com/google/error_analyzer/data/constant/FileConstants.java

```diff
@@ -14,4 +14,6 @@
 /** This class contains all the constants related to files*/
 public class FileConstants {
     public static final String TEXT_HTML_CONTENT_TYPE = "text/html";
+    public static final String APPLICATION_JSON_CONTENT_TYPE = 
```

Comment:
Already present in master. Please sync branch to master.

Reply:
I have raised this PR to StoreUrlLogs branch, will sync while merging to master

Reply:
Ideally, we should sync StoreUrlLogs to master.
Then we should sync encode to StoreUrlLogs.